Changelog.md

```diff
@@ -5,6 +5,7 @@ Language Features:
 
 Compiler Features:
 * LSP: Add rudimentary support for semantic highlighting.
+* TypeChecker: Support using foreign constants as compile-time constants
```

Comment:
I think this wording might be a little clearer to users:
```suggestion
* TypeChecker: Support using library constants in initializers of other constants.
```

Reply:
Agree, done.